pkg/settings/setting.go

```diff
@@ -15,6 +15,7 @@ var (
 	CLIURLDarwin                    = NewSetting("cli-url-darwin", "https://releases.rancher.com/cli/v1.0.0-alpha8/rancher-darwin-amd64-v1.0.0-alpha8.tar.gz")
 	CLIURLLinux                     = NewSetting("cli-url-linux", "https://releases.rancher.com/cli/v1.0.0-alpha8/rancher-linux-amd64-v1.0.0-alpha8.tar.gz")
 	CLIURLWindows                   = NewSetting("cli-url-windows", "https://releases.rancher.com/cli/v1.0.0-alpha8/rancher-windows-386-v1.0.0-alpha8.zip")
+	ClusterDefaults                 = NewSetting("cluster-defaults", "")
```

Comment:
@kinarashah we should include the basic config. It can carry the current schema defaults - no need to include all the fields, just the most important ones

Reply:
have added function such that cluster-defaults would have values from current schema on startup/reload